pkg/ebpf/c/tracee.bpf.c

```diff
@@ -1391,28 +1392,48 @@ static __always_inline int get_iface_config(int ifindex)
     return *config;
 }
 
+// INTERNAL: STRINGS -------------------------------------------------------------------------------
+
+static __inline int has_prefix(char *prefix, char *str, int n)
+{
+    int i;
+#pragma unroll
```

Comment:
I don't know why this happens in clang-14, but it actually gave me an idea.
Instead of comparing char by char like has_prefix does, we can make the comparison by casting to u64*, and compare the 16 bytes by two u64 comparisons.